qa-tests-backend/src/main/groovy/services/BaseService.groovy

```diff
@@ -104,41 +104,43 @@ class BaseService {
         }
     }
 
-    static ManagedChannel transportChannel = null
-    static ClientInterceptor authInterceptor = null
-    static Channel effectiveChannel = null
-    private static boolean useClientCert = false
+    // codenarc-disable FieldName
```

Comment:
Why not naming it as codenarc proposes?

Reply:
Could you add `@CompileStatic` for this class?

```
BaseService.groovy: 113: [Static type checking] - Incompatible generic argument types. Cannot assign java.lang.ThreadLocal<java.lang.Boolean> to: java.lang.ThreadLocal<boolean>
```

Reply:
> Why not naming it as codenarc proposes?

codenarc is wrong :). They should not be named like constants as it suggests.

Reply:
actually if it's `private static final` it's constant reference although you can change what is inside :)